tests/lib/muinstaller/main.go

```diff
@@ -559,21 +583,25 @@ func run(seedLabel, rootfsCreator, bootDevice string) error {
 }
 
 func main() {
-	if len(os.Args) != 4 {
-		// XXX: allow installing real UC without a classic-rootfs later
-		fmt.Fprintf(os.Stderr, "Usage: %s <seed-label> <rootfs-creator> <target-device>\n", os.Args[0])
+	if len(os.Args) < 3 || len(os.Args) > 4 {
+		fmt.Fprintf(os.Stderr, "Usage: %s <seed-label> <target-device> [rootfs-creator]\n"+
+			"If [rootfs-creator] is specified, classic Ubuntu with core boot will be installed.\n"+
+			"Otherwise, Ubuntu Core will be installed\n", os.Args[0])
 		os.Exit(1)
 	}
 	logger.SimpleSetup()
 
 	seedLabel := os.Args[1]
-	rootfsCreator := os.Args[2]
-	bootDevice := os.Args[3]
+	bootDevice := os.Args[2]
+	rootfsCreator := ""
+	if len(os.Args) > 3 {
+		rootfsCreator = os.Args[3]
+	}
 	if bootDevice == "auto" {
 		bootDevice = waitForDevice()
 	}
 
-	if err := run(seedLabel, rootfsCreator, bootDevice); err != nil {
+	if err := run(seedLabel, bootDevice, rootfsCreator); err != nil {
```

Comment:
Is there a particular reason that the order of `bootDevice` and `rootfsCreator` were switched?

Reply:
Just to keep consistency with the new expected order of arguments to muinstaller